src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -141,4 +141,20 @@ public String getDisplayNameGiver() {
     public String getDisplayNameRecipient() {
         return displayNameRecipient;
     }
+    
+    public static List<FeedbackParticipantType> getParticipantListFromParams(String participantListParams) {
```

Comment:
Also, `participantListParams` may not be a good parameter name. `params` is only relevant when `HttpServletRequest` is involved and this class is independent of that concept.
Something like `getParticipantListFromCommaSeparatedValues` is better as it is independent of the concept of `params`, albeit it may be too long for a name.